### PR TITLE
[ACR-1343] Add a new feature description for thermal API

### PR DESCRIPTION
**Pull request by lokilee73**
Add a new feature description for thermal API
Signed-off-by: lokilee73 <changjoo.lee@samsung.com>

### Change Description ###

 - ACR: http://suprem.sec.samsung.net/jira/browse/ACR-1343

### CODE REVIEW

**Comment by mijin85cho** [comment]
@surbhig12 Please review the updated "thermistor" part only.

File: docs/application/native/guides/device/system.md [hunk 1]
```diff
@@ -644,6 +645,17 @@ The following table lists the system setting feature keys:
...
 | `http://tizen.org/feature/systemsetting.notification_email` | `bool` | The platform returns `true` for this key and the `http://tizen.org/feature/systemsetting` key, if the device supports a way to change or get a ringtone for all email notifications. |
 
+The following table lists the thermistor feature keys:
+
+<a name="thermistor"></a>
+**Table: Thermistor feature keys**
+
+| Key                                      | Type   | Description                              |
+|------------------------------------------|--------|------------------------------------------|
+| `http://tizen.org/feature/thermistor.ap` | `bool` | The platform returns `true` for this key, if the device supports a thermistor for Application Processor. |
```
Comment (surbhig12) [inline docs/application/native/guides/device/system.md]: Suggestion: To maintain consistency in the page, please rephrase to:

The platform returns `true` for this key, if the device supports a thermistor for the application processor.

I have added "the" as processor here is a qualifier. Also,  in the "Table: Platform system keys" section, we have used sentence case.

File: docs/application/native/guides/device/system.md [hunk 2]
```diff
@@ -644,6 +645,17 @@ The following table lists the system setting feature keys:
...
+| `http://tizen.org/feature/thermistor.cp` | `bool` | The platform returns `true` for this key, if the device supports a thermistor for Communications Processor. |
```
Comment (surbhig12) [inline docs/application/native/guides/device/system.md]: Suggestion: To maintain consistency in the page, please rephrase to:

The platform returns `true` for this key, if the device supports a thermistor for the communications processor.

I have added "the" as processor here is a qualifier. Also, in the "Table: Platform system keys" section, we have used sentence case.

File: docs/application/native/guides/device/system.md [hunk 3]
```diff
@@ -644,6 +645,17 @@ The following table lists the system setting feature keys:
...
+| `http://tizen.org/feature/thermistor.battery` | `bool` | The platform returns `true` for this key, if the device supports a thermistor for Battery. |
```
Comment (surbhig12) [inline docs/application/native/guides/device/system.md]: Suggestion: To maintain consistency in the page, please rephrase to:

The platform returns `true` for this key, if the device supports a thermistor for the battery.

I have added "the" as it explains that it refers to the battery specific to the device. Also, in the "Table: Battery feature keys" section, we have used sentence case.

**Comment by lokilee73** [comment]
close to update again

**Pull request closed without merging**